modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -6,51 +6,118 @@
 [id="talo-platform-prepare-end-to-end_{context}"]
 = End-to-end procedures for updating clusters in a disconnected environment
 
+If you have deployed spoke clusters with distributed unit (DU) profiles using the GitOps ZTP with Topology Aware Lifecycle Operator pipeline described in "Deploying distributed units at scale in a disconnected environment", this procedure describes how to upgrade your spoke clusters and Operators.
+
 [id="talo-platform-prepare-for-update_{context}"]
 == Preparing for the updates
 
-If both the hub and the spoke clusters are running {product-title} 4.9, you need to update ZTP from version 4.9 to 4.10. If {product-title} 4.10 is used, you can set up the environment.
+If both the hub and the spoke clusters are running {product-title} 4.9, you must update ZTP from version 4.9 to 4.10. If {product-title} 4.10 is used, you can set up the environment.
 
 [id="talo-platform-prepare-for-update-env-setup_{context}"]
 == Setting up the environment
 
-If you deployed ZTP in a disconnected environment as instructed in the "Preparing the disconnected environment" section in "Deploying distributed units at scale in a disconnected environment", you need to do the following steps before TALO can successfully perform update operations:
+The TALO can perform both platform and Operator updates.
 
-TALO can perform both platform and Operator updates:
+You must mirror both the platform image and Operator images that you want to update to in your mirror registry before you can use the TALO to update your disconnected clusters. Complete the following steps to mirror the images:
 
-* For platform updates, you have to do the following:
+* For platform updates, you must perform the following steps:
 +
 . Mirror the desired {product-title} image repository. For more information about how to mirror an image repository, see the "Mirroring the OpenShift Container Platform image repository" section.
+
+. Save the output of the `imageContentSources` section in a file. Retrieve the contents of the `imageContentSources` section, by running the following command:
 +
-[IMPORTANT]
-====
-The `imageContentSources` section, which can be retrieved with the `oc adm release mirror` command, needs to be saved for platform updates.
-For example, this is the output of the `imageContentSources` which contains the information of your mirrors:
+[source,terminal]
+----
+$ oc adm release mirror
+----
 +
+.Example output
+
 [source,terminal]
 ----
 imageContentSources:
-       - mirrors:
-         - registry.example.com:5000/openshift4
-         source: quay.io/openshift-release-dev/ocp-v4.0-art
+ - mirrors:
+   - mirror-ocp-registry.ibmcloud.io.cpak:5000/openshift-release-dev/openshift4
+   source: quay.io/openshift-release-dev/ocp-release
+ - mirrors:
+   - mirror-ocp-registry.ibmcloud.io.cpak:5000/openshift-release-dev/openshift4
+   source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
 ----
++
+[NOTE]
```

Comment:
This should not be under "Save the output of the `imageContentSources` section in a file ... ", they are in parallel. From this line to line 104, the steps are about how to get the image signature of an ocp image.